kpi/models/import_export_task.py

```diff
@@ -342,9 +343,11 @@ class ExportTask(ImportExportTask):
     `data` attribute to a dictionary with the following keys:
     * `type`: required; `xls` or `csv`
     * `source`: required; URL of a deployed `Asset`
-    * `lang`: optional; `xml` for XML names or the name of the language to be
-              used for labels. Leave unset, or use `_default` or `None`, for
-              labels in the default language
+    * `lang`: optional; It can be:
+        - unset (`None`) or `formpack.constants.UNTRANSLATED` for labels in the default language (default: `None`).
```

Comment:
I try to keep lines under 80 characters for readability. It can certainly be a pain sometimes.

Is there a docstring parser you like to use that doesn't handle line breaks within these parameter descriptions well?